Driverless AI/getting-started-with-driverless-ai-test-drive/test-drive.md

```diff
@@ -77,16 +77,26 @@ If you are new to Driverless AI we highly recommend you start with the **Test Dr
 
 ## How to Start and End the Test Drive Lab Session
 
-After a successful login the Driverless AI Test Drive Lab will appear:
+After a successful login, select the "View Details" button on lab one, as shown in the image below. Ignore other labs as we will only be working on lab one. 
```

Comment:
Can you please add the following:

After a successful login, select "Browse Labs" located on the top-left corner of the page, then Select **Lab 1- Driverless AI Test Drive (x.x.x.x LTS) (1GPU)**.

Click on the "View Details" button on lab one, as shown in the image below.